wasm2map/src/test.rs

```diff
@@ -24,6 +24,21 @@ fn can_create_sourcemap() {
     });
 }
 
+#[test]
+fn relative_paths() {
+    testutils::run_test(|out| {
+        if let Ok(mapper) = WASM::load(&out) {
+            let sourcemap = mapper.map_v3();
+
+            // Any fixed relative path should have at least a `/` beforehand.
+            assert!(sourcemap.contains("/library/core/src/any.rs"));
+            assert!(sourcemap.contains("/library/core/src/panicking.rs"));
+        } else {
+            unreachable!()
+        }
+    });
+}
```

Comment:
I'm happy to take any suggestions on how to test this otherwise.

Reply:
I think it's okay for now, the code path you added runs during the tests, and PR #10 will be the more significant one.